includes/azure-app-configuration-availability-zones.md

```diff
@@ -15,7 +15,8 @@ Following are regions where App Configuration has enabled availability zone supp
 | Canada Central   | Germany West Central | UAE North     |        | Central India  |
 | Central US       | North Europe         |               |        | Japan East     |
 | East US          | Norway East          |               |        | Korea Central  |
-| East US 2        | UK South             |               |        | Southeast Asia |
-| South Central US | West Europe          |               |        | East Asia      |
-| West US 2        | Sweden Central       |               |        | China North 3  |
-| West US 3        |                      |               |        |                |
+| East US 2        | Poland Central       |               |        | Southeast Asia |
+| South Central US | Sweden Central       |               |        | East Asia      |
+| US Gov Virginia  | Switzerland North    |               |        | China North 3  |
```

Comment:
We support AZ in FFV. The storage accounts are not migrated to ZRS yet.

Reply:
This has been updated to remove FFV. Thanks!